src/main/java/net/mcreator/util/image/ImageUtils.java

```diff
@@ -82,6 +82,20 @@ public static ImageIcon drawOver(ImageIcon i, ImageIcon wh) {
 		return new ImageIcon(resizedImage);
 	}
 
+	public static ImageIcon drawOver(ImageIcon i) {

```

Comment:
Overload existing method, do not make duplicate code

Reply:
Done!